dbt/adapters/bigquery/connections.py

```diff
@@ -15,7 +15,7 @@
 
 import google.auth
 import google.auth.exceptions
-import google.cloud.bigquery as bigquery
+import google.cloud.bigquery
```

Comment:
How were you able to rename this without changing any references (besides the removed line) and still have it be used? I'm surprised the linter didn't pick it up as imported but not used.